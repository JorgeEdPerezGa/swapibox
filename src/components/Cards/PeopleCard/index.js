import React from 'react';

const PeopleCard = ({ name, birthYear, eyeColor, height, mass, skinColor, homeworld, homeworldPopulation }) => {
  return (
    <article className = 'card'>
      <h1>{ name }</h1>
      <p>birth year: { birthYear }</p>
      <p>eye color: { eyeColor }</p>
      <p>height: { height }</p>
      <p>mass: { mass }</p>
      <p>skin color: { skinColor }</p>
      <p>homeworld: { homeworld }</p>
      <p>homeworld population: { homeworldPopulation }</p>
    </article>
  );
};

export default PeopleCard;
