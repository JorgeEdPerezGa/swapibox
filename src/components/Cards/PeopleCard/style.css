p {
  margin-left: 20px; }

.card {
  width: 300px;
  background: white;
  margin: 30px;
  text-align: left; }

.card-title {
  color: white;
  margin: 0;
  padding: 30px;
  text-align: center; }

.card-title-person {
  background: #420606; }
