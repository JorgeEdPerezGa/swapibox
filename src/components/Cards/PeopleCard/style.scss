p {
  margin-left: 20px;
}

.card {
  width: 300px;
  background: white;
  margin: 30px;
  text-align: left;
}

.card-title {
  color: white;
  margin: 0;
  padding: 25px;
  font-weight: 300;
  text-align: center;
}

.card-title-person {
  background: #420606;
  padding-right: 50px;
}
