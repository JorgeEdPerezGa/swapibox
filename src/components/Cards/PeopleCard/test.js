import React from 'react';
import PeopleCard from './index';
import setupTest from '../../../setupTests';
import { shallow } from 'enzyme';

describe('<PeopleCard/>', () => {
  const wrapper = shallow(<PeopleCard />);

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
