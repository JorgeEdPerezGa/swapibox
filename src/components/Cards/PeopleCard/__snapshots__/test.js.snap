// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PeopleCard/> should match snapshot 1`] = `<PeopleCard />`;
